modules/aws/cloudwatch.go

```diff
@@ -56,3 +57,21 @@ func NewCloudWatchLogsClientE(t testing.TestingT, region string) (*cloudwatchlog
 	}
 	return cloudwatchlogs.New(sess), nil
 }
+
+// NewCloudWatchClient creates a new CloudWatch client.
+func NewCloudWatchClient(t testing.TestingT, region string) *cloudwatch.CloudWatch {
+	client, err := NewCloudWatchClientE(t, region)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
NIT: we've started using `require.NoError(t, err)` to reduce these 3 lines down to 1.

Reply:
Sounds good - I've updated those blocks to use `require.NoError(t, err)`.